src/screens/LoginScreen.js

```diff
@@ -22,20 +34,66 @@ const styles = StyleSheet.create({
   },
   caption: {
     marginTop: 10,
+    marginRight: 12,
```

Comment:
Nit: increase left and right margins for the caption. Figma suggests 40px on each side, prob ~20 would be good